archetype/test/setup.js

```diff
@@ -3,8 +3,8 @@
 const TrailsApp = require('trails')
 
 before(() => {
-  global.app = new TrailsApp(require('../../'))
-  return global.app.start()
+  global.app = new TrailsApp(require('../'))
+  return global.app.start().catch(global.app.stop)
```

Comment:
I added this logic because I saw other trailpacks doing the same.

All the errors I tried to cause on app load caused the tests to timeout, without actually triggering that catch.

I can remove the `catch`. Would that be better?